modules/hcp-updates-hosted-cluster.adoc

```diff
@@ -9,3 +9,8 @@
 The `spec.release` value dictates the version of the control plane. The `HostedCluster` object transmits the intended `spec.release` value to the `HostedControlPlane.spec.release` value and runs the appropriate Control Plane Operator version.
 
 The hosted control plane manages the rollout of the new version of the control plane components along with any {product-title} components through the new version of the Cluster Version Operator (CVO).
+
+[IMPORTANT]
+====
+In {hcp}, the `NodeHealthCheck` resource cannot detect the status of the CVO. A cluster administrator must pause the node health check remediation manually before performing critical operations such as updating the cluster, to prevent new remediation actions from interfering with cluster updates. To pause the node health check remediation, set the value of the `pause` field to `true` in the `NodeHealthCheck` resource. After the cluster update is complete, resume the node health check remediation by setting the value of the `pause` field to `false`.
```

Comment:
I have not tested the NodeHealthCheck operator before, not sure about the part of setting value pause to true